Add blog post: Introducing YakShaver Anywhere (#760)
SSWConsulting/SSW.Products: .claude/skills/yakshaver-blog-banner/SKILL.md

---
name: yakshaver-blog-banner
description: Create a YakShaver blog banner image (1200×675 PNG) in the consistent house style — dark background, red glow, YakShaver logo, bold headline left, flat vector illustration right. Use whenever adding or updating a banner for a YakShaver blog post in content/blogs/YakShaver/.
---

# YakShaver Blog Banner Generator

Generates blog banner PNGs that match the existing YakShaver blog style. Banners are
authored as a self-contained HTML file, rendered to PNG with headless Chrome, and
saved to `public/YakShaver/Blogs/<blog-slug>.png`.

Before designing, look at 2–3 existing banners for calibration:
`public/YakShaver/Blogs/ROI.png`,

> `macos_update.png`, `yakshaver-anywhere.png` (view them with the Read tool).

Reviewer: The banner file was named with a `-banner` suffix, so this self-reference is broken on arrival:

```suggestion
`macos_update.png`, `yakshaver-anywhere-banner.png` (view them with the Read tool).
```

> `new-support-macos.png`, `yakshave-on-browser.png` (view them with the Read tool).

Reviewer: These calibration files don't exist — the banners on main are `ROI.png`, `macos_update.png`, `getting_started.png`, etc.:

```suggestion
`public/YakShaver/Blogs/ROI.png`,
`macos_update.png`, `yakshaver-anywhere.png` (view them with the Read tool).
```

- Give containers `display: flex; flex-direction: column` and inner areas `flex: 1` so nothing
  clips at fixed heights (this has bitten before)

## Process

1. **Copy the template** `template.html` (next to this file) into the scratchpad directory as
   `<blog-slug>-banner.html`.
2. **Set the logo path**: replace `{{LOGO_SRC}}` with the absolute file URL to the logo, e.g.
   `file:///<repo-root>/public/YakShaver/yakshaver-logo.png`.
3. **Write the copy**: kicker, headline (≤ 2 lines), subtitle. Update `<title>` too.
4. **Design the illustration**: replace the contents of `<div class="illustration">` with your
   concept, following the illustration rules above.
5. **Render** with headless Chrome from the scratchpad directory:

   ```bash
   "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" \
     --headless=new --disable-gpu --hide-scrollbars \
     --window-size=1200,675 \
     --screenshot=<blog-slug>.png \
     --allow-file-access-from-files \
     "file://$PWD/<blog-slug>-banner.html"
   ```

   (On Windows/Linux substitute the Chrome/Chromium binary path.)
6. **Verify visually**: open the PNG with the Read tool and check —

>    - [ ] Exactly 1200×675 (`sips -g pixelWidth -g pixelHeight <file>.png`)

Reviewer: `sips` is macOS-only — worth a cross-platform note since the Chrome step already has one:

```suggestion
   - [ ] Exactly 1200×675 (macOS: `sips -g pixelWidth -g pixelHeight <file>.png`; Windows: `Add-Type -AssemblyName System.Drawing; [System.Drawing.Image]::FromFile("<file>.png") | Select Width, Height`)
```
